go-controller/pkg/ovn/gress_policy.go

```diff
@@ -97,6 +97,28 @@ func (gp *gressPolicy) ensurePeerAddressSet(factory addressset.AddressSetFactory
 	return nil
 }
 
+func (gp *gressPolicy) addPeerSvcVip(service *v1.Service) error {
+	if gp.peerAddressSet == nil {
+		return fmt.Errorf("peer AddressSet is nil, cannot add peer Service: %s for gressPolicy: %s",
+			service.ObjectMeta.Name, gp.policyName)
+	}
+	ip := getSvcVips(service)
+
+	klog.V(5).Infof("Adding SVC clusterIP to gressPolicy's Address Set: %v", ip)
+	return gp.peerAddressSet.AddIPs(ip)
+}
+
+func (gp *gressPolicy) deletePeerSvcVip(service *v1.Service) error {
+	if gp.peerAddressSet == nil {
+		return fmt.Errorf("peer AddressSet is nil, cannot add peer Service: %s for gressPolicy: %s",
+			service.ObjectMeta.Name, gp.policyName)
+	}
+	ip := getSvcVips(service)
+
+	klog.V(5).Infof("Deleting SVC clusterIP from gressPolicy's Address Set: %v", ip)
```

Comment:
this can be regular log level, just fix the log message to say something like:
klog.Infof("Deleting service %s, clusterIP: %s,  from gressPolicy's %s, Address Set: %s, <servcie name>, <ip>, <policy name>, <address set name>)